app/assets/javascripts/angular_modules/module_dialog_editor.js

```diff
@@ -0,0 +1,7 @@
+miqHttpInject(
```

Comment:
This is dead code if you define everything on `ManageIQ.angular.app`. Unless you want to make these dialogEditor dependencies explicitly visible, I think you can just remove this file..